.cr/personal/FavoritesList/List.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not sure what is `.cr/personal/FavoritesList/List.xml` but consider removing it or add it to a `.gitignore`

Reply:
Deleted file and added to .gitignore